ui-account/src/main/java/app/tivi/account/AccountUi.kt

```diff
@@ -86,7 +83,7 @@ fun AccountUi(
             ) {
                 @Suppress("DEPRECATION") // TODO: Migrate away from FlowRow
                 FlowRow(
-                    mainAxisAlignment = FlowMainAxisAlignment.End,
+                    mainAxisAlignment = androidx.compose.foundation.layout.FlowMainAxisAlignment.End,
```

Comment:
Import

Reply:
Nah, otherwise I need to suppress the whole file which I don't want to do. Using the full class name means I can scope the suppression to just here.